mct_quantizers/pytorch/quantizers/activation_inferable_quantizers/activation_symmetric_inferable_quantizer.py

```diff
@@ -88,7 +88,8 @@ def __init__(self,
             assert len(
                 self.scales) == 1, f'For activation, quantization per channel is not supported and threshold should ' \
                                    f'be of length 1 but is {len(threshold)}'
-            self.scales = self.scales[0]
+            
+            self.scales = float(self.scales[0])
```

Comment:
I hope this won't create issues for the converter.
Can you run it quickly by Chizky or is it too negligible to start explaining?

Reply:
It shouldn't be a problem since it caused issues only when we deserialized a torch fx graph (which is not the case for the converter

Reply:
I checked an exported model with Chizky through the converter, which passed.